HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
+  and mirror nodes. Historically, accounts created via these wallets are `ED25519` based and usually identify the
+  account via the `Hedera Account Id`.
+- `Hollow account` - a partially created account which lacks a public key but does have a valid `shard.realm.num`
+  accountId _and_ is auto-created with an `evm address alias`. Such an account may receive assets but may not take
+  part in transactions that require its signature, until it is promoted to being a "full" account.
+- `auto-create` - an account creation flow in which the `key alias` or `evm address alias` that a crypto-transfer
+  is addressed to is used to automatically create a new account (if none exists) without an explicit `CryptoCreate`
+  transaction being submitted. We call it `hollow-create` when the `evm address alias` is used, since it results
+  in the creation of a `Hollow account`.
+
+With an Ethereum-native wallet, you can transfer the native currency (in our case, HBAR), or tokens, to an `evm address`
+that does not yet exist. If nobody in the universe has the private key for that account, the value held by that account
+is effectively locked forever. If somebody does have the keys to that account, they can access the account and control
+it. Since Ethereum-native wallets do not understand the `0.0.10` style format of a Hedera account ID, it is necessary
+to support the creation of accounts with an `evm address alias` to enable compatibility between Ethereum-native
+wallets and Hedera-native wallets and all accounts.
 
 ## User stories
 
-1. As a user with an `ECDSA` based account from another chain I would like to create my Hedera account using my `public-address`.
-
-2. As a developer, I would like to `auto-create` a new account using a public key via the `CryptoCreate` transaction.
-
-3. As a developer, I would like to `lazy-create` a new account using a `public-address` via the `CryptoCreate` transaction.
+1. As a user with an `ECDSA` derived account from another chain, I would like to create my Hedera account using that
+   same ECDSA public key and associated Ethereum compatible `evm address alias`.
+2. As a developer, I would like to create a new account with a `key alias` via the `CryptoCreate` transaction.
+3. As a developer, I would like to create a new account with an `evm address alias` via the `CryptoCreate` transaction.
+4. As a developer, I would like to transfer HBAR or tokens to a Hedera account using their `evm address alias`.
+5. As a Hedera user with an `Ethereum-native wallet`, I would like to receive HBAR or tokens in my account by sharing
+   only my `evm address alias`.
+6. As a Hedera user with a `Hedera-native wallet`,  I would like to transfer HBAR or tokens to another account using
+   only the recipient's `evm address alias`.
+7. As a Hedera user with an `ED25519` key on my account, I want to add an `evm address alias` to my account.
```

Comment:
I think actually you would be OK using an Ethereum wallet and JSON-RPC relay if you use your long-zero address, AS LONG AS the contract you are calling isn't using `ecrecover`. If it is, then I agree, it won't work. For HIP-583, I think we accept that limitation, with the workaround being, you have to create a new account 😭 . But, since updating an existing account introduces a number of new wrinkles, including new work on the wallets, I think it is better to live with this limitation for the moment in order to onboard more users to Hedera and work on this in a new HIP.

Reply:
You can't work with an Ethereum wallet and JSON-RPC relay without signing with an ECDSA key that can be recovered and the resulting EVM address matched to an existing alias belonging to an account in Hedera.